thoth/package_extract/image.py

```diff
@@ -464,6 +465,17 @@ def construct_rootfs(dir_path: str, rootfs_path: str) -> list:
     return layers
 
 
+def _get_absolute_link(root_path: str, path: str, iter: int) -> str:
```

Comment:
```python
from typing import Optional

...

def _get_absolute_link(root_path: str, path: str, iter: int) -> Optional[str]:
...
```

Reply:
BTW import of `stat` is unused in this module.

Reply:
It looks like it is being used at https://github.com/thoth-station/package-extract/blob/master/thoth/package_extract/image.py#L151